components/TextFields/src/MDCTextField.m

```diff
@@ -523,6 +523,7 @@ - (CGRect)textRectForBounds:(CGRect)bounds {
                                                     self.placeholderLabel.font.lineHeight) /
                                                 2.f);  // Text field or placeholder
   actualY = textInsets.top - actualY;
+  actualY += 1.f;
```

Comment:
Super-nit: Consider adding "+ 1." to the previous line.

Reply:
Done. and made a const.